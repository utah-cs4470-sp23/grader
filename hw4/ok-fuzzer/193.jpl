time write image .79 { 8 } to ""
write image 76 to ""
show .987
fn r_2 ( { {} , {} } ) : H. { 
 return .4 
 let {} = false[] 
 return 282.4 { 661 } 
 assert U.8_886 ( ) { 74 }, "" 
 }
assert false, "&@"
fn Z._F9 ( { { { {} } , {} , {} } } ) : m { 
  
 }
time time let {} = v
assert {}, "="
show ( [] )
let Nd = 83
show J_.9 ( )
read image "{" to t5..933[]
write image [ [] ] to ""
show 5[]
fn k1 ( { { { {} } } , X._ : { int , bool , {} } } ) : int { 
  
 }
write image 2 to "?&+{}"
show [ false ]
assert 07, ""
