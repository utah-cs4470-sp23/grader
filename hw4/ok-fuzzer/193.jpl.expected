(TimeCmd (WriteCmd (TupleIndexExpr (FloatExpr 0) 8) ""))
(WriteCmd (IntExpr 76) "")
(ShowCmd (FloatExpr 0))
(FnCmd
 r_2
 ((TupleBinding (TupleBinding) (TupleBinding)))
 (VarType H.)
 (ReturnStmt (FloatExpr 0))
 (LetStmt (TupleLValue) (ArrayIndexExpr (FalseExpr)))
 (ReturnStmt (TupleIndexExpr (FloatExpr 282) 661))
 (AssertStmt (TupleIndexExpr (CallExpr U.8_886) 74) ""))
(AssertCmd (FalseExpr) "&@")
(FnCmd
 Z._F9
 ((TupleBinding
   (TupleBinding (TupleBinding (TupleBinding)) (TupleBinding) (TupleBinding))))
 (VarType m))
(TimeCmd (TimeCmd (LetCmd (TupleLValue) (VarExpr v))))
(AssertCmd (TupleLiteralExpr) "=")
(ShowCmd (ArrayLiteralExpr))
(LetCmd (ArgLValue (VarArgument Nd)) (IntExpr 83))
(ShowCmd (CallExpr J_.9))
(ReadCmd "{" (ArrayArgument t5..933))
(WriteCmd (ArrayLiteralExpr (ArrayLiteralExpr)) "")
(ShowCmd (ArrayIndexExpr (IntExpr 5)))
(FnCmd
 k1
 ((TupleBinding
   (TupleBinding (TupleBinding (TupleBinding)))
   (VarBinding (VarArgument X._) (TupleType (IntType) (BoolType) (TupleType)))))
 (IntType))
(WriteCmd (IntExpr 2) "?&+{}")
(ShowCmd (ArrayLiteralExpr (FalseExpr)))
(AssertCmd (IntExpr 7) "")
